littleblog/entry/views.py

```diff
@@ -20,37 +23,17 @@ class EntryViewSet(viewsets.ModelViewSet):
     queryset = Entry.objects.all()
     permission_classes = [IsAuthenticated, IsOwnerOrReadOnly]
 
+    @action(detail=True, methods=['patch'])
+    def rate(self, request, pk=None):
+        instance = self.get_object()
+        serializer = RatingUpdateSerializer(instance, data=request.data, partial=True)
+        if serializer.is_valid(raise_exception=True):
```

Comment:
no need to do `if` when you use `raise_exception=True`.

It makes little sense, because when an exception is raised - the code won't go further anyway. ie, this code works same, but more straightforward:
```
serializer.is_valid(raise_exception=True)
set_mark(instance, **serializer.validated_data)
```